migrations/db/20201111164339_cron-command_update_cron_user.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace EE\Migration;
+
+use EE;
+use EE\Migration\Base;
+use EE\Model\Cron;
+
+class UpdateCronUser extends Base {
+
+
```

Comment:
```suggestion
	public function __construct() {

		parent::__construct();
		$this->sites = Site::all();
		if ( $this->is_first_execution || ! $this->sites ) {
			$this->skip_this_migration = true;
		}
	}
```

Reply:
`$this->skip_this_migration` that has been used in `up()` function is not initialized, need to extend constructor do it as in suggestion.